src/cli/spvnode.c

```diff
@@ -386,6 +421,7 @@ int main(int argc, char* argv[]) {
     char* address = NULL;
     dogecoin_bool use_checkpoint = false;
     char* mnemonic_in = 0;
+    char* name = 0;
     dogecoin_bool full_sync = false;
     dogecoin_bool have_decl_daemon = false;
 
```

Comment:
Think you might be missing the `w` in the `getopt_long_only` function on L436. Have tested using the same routine as #131 and it works but no wallet files were created in running:

1)
```
cp main_headers_april25.db main_headers.db && rm main_wallet.db && make && spvnode -c -a "D6JQ6C48u9yYYarubpzdn2tbfvEq12vqeY DGYrGxANmgjcoZ9xJWncHr6fuA6Y1ZQ56Y DBcR32NXYtFy6p4nzSrnVVyYLjR42VxvwR" -b -p scan
```
2)
```
cp main_headers_june5.db main_headers.db && make && spvnode -c -b -p scan
```
3)
```
make && spvnode -c -b -p scan
```

Is there a recommended approach for testing this?

Reply:
wallet address files*